mergify/src/main/scala/org/typelevel/sbt/mergify/MergifyAction.scala

```diff
@@ -57,6 +59,21 @@ object MergifyAction {
       }
   }
 
+  final case class RequestReviews(users: List[String] = Nil) extends MergifyAction {
```

Comment:
Can we also add the `teams` and `users_from_teams` keys?. Also frustratingly seems like all of these can be either a `List` or a `Map`. I wonder if we should just go with the `Map` since it subsumes `List` and maybe add some helpful constructors.

https://docs.mergify.com/actions/request_reviews/

I know it's annoying, but fixing these binary-compatibly will be even more annoying :)

Reply:
Good idea, I'll play with this tomorrow or later this weekend.

Reply:
Had some extra time tonight after all. WDYT of what I just pushed up?

(I wondered whether any of these should be case classes if we're worried about bincompat, btw. Should I change the others too?)